spec/requests/api/articles/index_spec.rb

```diff
@@ -9,7 +12,16 @@
 
   it { is_expected.to have_http_status 200 }
 
-  it 'is expected to return a collection of 5 articles' do
-    expect(response_json['articles'].size).to eq 5
+  # it 'is expected to return a collection of 5 articles' do
+  #   expect(response_json['articles'].size).to eq 5
+  # end
```

Comment:
Why is this commented out?

Reply:
That is left after Tomas's session:
https://youtu.be/LT24ryaqlFQ?t=851

Should we delete?

Reply:
deleted